libs/scripts/delayed.js

```diff
@@ -28,7 +28,9 @@ export const loadPrivacy = async (getConfig, loadScript) => {
   const acom = '7a5eb705-95ed-4cc4-a11d-0cc5760e93db';
   const ids = {
     'hlx.page': '3a6a37fe-9e07-4aa9-8640-8f358a623271-test',
+    'aem.page': '3a6a37fe-9e07-4aa9-8640-8f358a623271-test',
     'hlx.live': '926b16ce-cc88-4c6a-af45-21749f3167f3-test',
+    'aem.live': '926b16ce-cc88-4c6a-af45-21749f3167f3-test',
```

Comment:
In my initial PR, I changed this a bit to not duplicate IDs, maybe it's worth considering - https://github.com/adobecom/milo/compare/helix-5?expand=1#diff-c21f6b181685a27721a54ce2b519823f6186157499b08ea290437d3fdff64473